### DISCUSSION TITLE
It is possible to transform a normal installation into PVE script install?

### DISCUSSION THREAD

**Discussion by platini76** [community-scripts/ProxmoxVE#500] [Q&A] [answered]
I have an old installation of spoolman, can I update it to a new script instllation to have update function and others without reistall it?

script is: bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/misc/update-repo.sh)"

**Comment by MickLesk** [comment]
This updates the link/path of the package to our repo. So you can use the “update” function again.

**Comment by platini76** [comment]
but if I launch this command on proxmox shell it will not install another ct ?  it will only link repo?

**Comment by platini76** [comment]
this is resutls:root@spoolman:~# upgrade
-bash: upgrade: command not found
root@spoolman:~# update
-bash: update: command not found
root@spoolman:~#

**Comment by MickLesk** [comment, marked as answer]
you need to run this on the main node. 

This does nothing other than replace tteck/Proxmox in the usr/bin/update of all LXC with our repo.
```bash
 "sed -i 's/tteck\\/Proxmox/community-scripts\\/ProxmoxVE/g' /usr/bin/update
```


if you dont have the update function (for whatever reason), execute this in your LXC, replace "APP" with your LXC Scriptname (f.e. spoolman)
```bash
echo "bash -c \"\$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/APP.sh)\"" >/usr/bin/update
chmod +x /usr/bin/update
```

**Comment by platini76** [comment]
thank you!

**Comment by platini76** [comment]
✗ No Spoolman Installation Found!
root@spoolman:~#

**Comment by MickLesk** [comment]
how you add spoolman? 
where is spoolman?

**Comment by platini76** [comment]
I don't know where is... is there a command to discover where is and update script?
I usually use this command to update:
docker compose pull && docker compose up -d

**Reply by MickLesk** [reply]
then its not an script from us :D 

Scripts from us are in /opt/APPNAME and the update is in usr/bin/update located.

---
Edit: 
Or is that what you mean by transfer?
I don't know where Spoolman docker stores its data. Nobody here will be able to answer that either

An fresh install is the better way to go

**Comment by platini76** [comment]
in fact the title was: It is possible to transform a normal installation into PVE script install?
so it is possible to update this installation to have a new script update function?

**Reply by MickLesk** [reply]
no

**Comment by platini76** [comment]
it is impossible? also a command to discover where it is installed?

**Reply by MickLesk** [reply]
its docker based, you can only check the docker-compose and Dockerfile and find your data. Then you need to check where the data located and export (like ftp) 

There is no script for this and there never will be.
